urllib3/connection.py

```diff
@@ -62,6 +57,14 @@ class ConnectionError(Exception):
 RECENT_DATE = datetime.date(2014, 1, 1)
 
 
+class DummyConnection(object):
+    "Used to detect a failed ConnectionCls import."
+    pass
+
+
+HTTPSConnection = DummyConnection
```

Comment:
I think we can get rid of this assignment, we override it twice more below anyways.